src/sentry/static/sentry/app/types/index.tsx

```diff
@@ -702,3 +702,26 @@ export type SelectValue<T> = {
   label: string;
   value: T;
 };
+
+/**
+ * The issue config form fields we get are basically the form fields we use in the UI but with some extra information.
+ * we can also assume that certain optional fields are required.
```

Comment:
When? why are the optional then?

Reply:
@EvanPurkhiser maybe they are option from the perspective of the UI field but they are guaranteed to exist from the data model used in the back end.